kANe/v1p1/01_meaning_of_dharma.md

```diff
@@ -3,72 +3,45 @@ title = "01 Meaning of Dharma"
-
-13 a tseitei : 19. 1. . 1. 1. 2. 14 अथातो धर्म व्याख्यास्यामः । यतोभ्युदयनिःश्रेयससिद्धिः स धर्मः । वैशेषिकसूत्र. 14a H II. 1. Page 1an: ....fatura i 15 अथातो धर्म व्याख्यास्यामः । श्रुतिप्रमाणको धर्मः । श्रुतिश्च द्विविधा वैदिकी तान्त्रिकी 
-
-च । quoted by कुल्लक on मनु 2. 1. 16 Vide Dr. Stcherbatsky's monograph 'the central conception of 
-
-Buddhism' (1923) p. 73. and' dharmas of the Buddbist'in I. H. Q. Vol. X. (1924) pp. 737-760 (at pp. 740 ff) by the same autbor. Vide * Buddhist conception of dbarma' by Prof. P. T. Raju in ABORI Vol. XXI pp. 192-202; " Essentials of Buddhist Philosophy" by Prof. J. Takakusu (1956) particularly pp. 106-111; also Dr. E. Conze's * Buddhist thought in India' (1962) pp. 92-103 for the different meanings of " Dharma and Dharmas" (elements of existence) in Buddbist works. 
-
-This is not the place to discuss what Asoka's Dhamma was, From tbe 4th Rock Edict it appears that 12 years after his coronation the Dhamma he wanted to spread or propagate comprised abstentio n injury to living creatures, courtesy for one's relatives and for H umanid 
-
-(Continued on next page) 
-
-History of Dharmaśāstra 
-
-The present work will deal with the sources of dharma, their contents, their chronology and other kindred matters. As the material is vast and the number of works is extremely large, only a few selected works and some important authors will be taken up for detailod treatment. More space will he dovotod to comparatively early works. 
+[^16]:
+    _Vide_ Dr. Stcherbatsky's monograph 'the central conception of Buddhism' (1923) p. 73. and 'dharmas of the Buddhist' in I. H. Q. Vol. X. (1924) pp. 737-760 (at pp. 740 ff) by the same autbor. Vide 'Buddhist conception of dharma' by Prof. P. T. Raju in ABORI Vol. XXI pp. 192-202; "Essentials of Buddhist Philosophy" by Prof. J. Takakusu (1956) particularly pp. 106-111; also Dr. E. Conze's 'Buddhist thought in India' (1962) pp. 92-103 for the different meanings of "Dharma and Dharmas" (elements of existence) in Buddhist works.
```

Comment:
There should be  an empty line before this. Please also reply to this comment once done so that I can merge.